Tests/test_PDB.py

```diff
@@ -1576,6 +1576,20 @@ def test_pdb_to_xyzr(self):
         assert len(msms_radii) == len(biopy_radii)
         self.assertSequenceEqual(msms_radii, biopy_radii)
 
+class FragmentMapperTests(unittest.TestCase):
+    """Tests for FragmentMapper module."""
+    
+    def self_test(self):
+        """Self test for FragmentMapper module."""
+        p = PDBParser()
+        pdb1= "PDB/1A8O.pdb"
+        s = p.get_structure("X", pdb1)
+        self.assertEqual(m, s[0])
+        fm = FragmentMapper(m, 10, 5, "levitt_data")
+        for r in Selection.unfold_entities(m, "R"):
+            self.assertEqual(str(r), "R")
+            if r in fm:
+                print(fm[r])
```

Comment:
As explained in the module docstring, these are available from http://csb.stanford.edu/~rachel/fragments/

If the code needs these files all present, we're not going to be able to do this via TravisCI. Rather we'd make the test conditional on the files being pre-downloaded (much like we skip tests if a binary tool is missing).

If the test case only needs on of these files, it may OK to download it as part of the test...